assets/css/solcast.webflow.style.css

```diff
@@ -664,20 +664,22 @@ table tbody tr:last-child td:last-child{
 }
 
 /* API Status Styling */
-
 .sp-status {
   position: relative;
   display: inline-block;
 }
+
+/* Status Badge Styles */
```

Comment:
@Rework-Digital - I take it this is the css from the files that are blocked because of the security policy.

Can you reference here that it came from https://statuspal.io/js/widget.js (I think) therefore:

1.  We're crediting appropriately 
2. When it changes or updates in a year's time, we'll know where to get the changed CSS from

Reply:
Added credit